includes/admin.inc

```diff
@@ -1994,6 +1994,13 @@ function islandora_solr_admin_settings_facet_fields($form, &$form_state, $variab
       'date_filter_datepicker_enabled' => 0,
       'date_filter_datepicker_range' => '-100:+3',
     );
+    $form['options']['date_facet_format'] = array(
+      '#type' => 'textfield',
+      '#title' => t('Date format'),
+      '#default_value' => $values['date_facet_format'],
+      '#description' => t('The format of the date, as it will be displayed in the facet block. Use <a href="!url">PHP date()</a> formatting.', array('!url' => 'http://lucene.apache.org/solr/api/org/apache/solr/util/DateMathParser.html')),
```

Comment:
That's actually been in there for years based on the copied block below. But a good catch nonetheless.